src/test/scala/mesosphere/marathon/raml/AppConversionTest.scala

```diff
@@ -108,7 +108,7 @@ class AppConversionTest extends UnitTest with ValidationTestLike {
       val protoRamlApp = app.toProto.toRaml[App]
 
       Then("The direct and indirect RAML conversions are identical")
-      val config = AppNormalization.Configuration(None, "bridge-name", Set(), ValidationHelper.roleSettings)
+      val config = AppNormalization.Configuration(None, "bridge-name", Set(), ValidationHelper.roleSettings, DeprecatedFeatureSet.empty(SemVer(1, 9, 0)))
```

Comment:
If this thing just received a Boolean, then we wouldn't have to construct this pretty unrelated object, and hardcode the current version of Marathon in our tests.

Reply:
Good point.